README.md

```diff
@@ -19,6 +21,13 @@ doxygen Doxyfile
 firefox doc/html/index.html
 ```
 
+###CMake Supported###
+
+Clone the project to some local path and add the following line to your `CMakeLists.txt`:
+
+    find_package(elegant-progressbar-for-a-more-civilized-age 1.0.0 REQUIRED PATHS <ABSOLUTE-PATH-TO-ELEGANT-PROGRESSBAR>)
```

Comment:
this should be plural (progressbar**s**), see name of the repository

There could (does not have to be) also be a comment that the `PATHS` argument is not required if you install to `/usr/lib`